test/integration/elrepo/inspec/elrepo_spec.rb

```diff
@@ -0,0 +1,32 @@
+arch = File.readlines('/proc/cpuinfo').grep(/POWER9/).any? ? 'power9' : os.arch
+case os.release.to_i
+
+when 7
+
+  # Test the elrepo repository
+  describe yum.repo('elrepo') do
+    if arch == 'x86_64'
+      it { should exist }
+    else
+      it { should_not exist }
+    end
+    it { should be_enabled }
+    its('baseurl') { should eq 'https://ftp.osuosl.org/pub/elrepo/elrepo/el7/x86_64/' }
```

Comment:
This test may not need separate cases for C7/C8:

```suggestion
    its('baseurl') { should eq "https://ftp.osuosl.org/pub/elrepo/elrepo/el#{os.release.to_i}/x86_64/" }
```